docs/ImJoy-Engine-Manager.imjoy.html

```diff
@@ -78,12 +78,14 @@
                 "inputType": "text",
                 "label": "Engine URL",
                 "model": "url",
+                "hint": "an secured url (starts with https:// ) to the plugin engine, the only unsecured exception in Chrome is localhost url(http://127.0.0.1:<PORT>)."
               },
               {
                 "type": "input",
                 "inputType": "text",
                 "label": "Connection Token",
                 "model": "token",
+                "hint": "a token you get from the plugin engine when you launch it."
```

Comment:
We shouldn't end with period of we don't start with capital letter, ie if it's not a real sentence.